tests/test_parsers.py

```diff
@@ -11,11 +15,38 @@ def test_parse_query_params() -> None:
         "polluting": False,
     }
     request = create_test_request(query=query)  # type: ignore
-    result = parse_query_params(connection=request)
+    result = parsers.parse_query_params(connection=request)
```

Comment:
i usually prefer to not use a namespace, i.e. `parse_query_params` rather than `parsers.parse_query_params`

Reply:
Happy to go with the flow

Reply:
When using namespaces, I like when things are named to read nicer/avoid repetition, e.g.,  `parse.query_params` rather than `parsers.parse_query_params`. 

All good though, just throwing in my 2c, use whatever naming you are happy with/get rid of the namespace altogether as @Goldziher suggests.